Jenkinsfile

```diff
@@ -11,18 +11,23 @@ elifePipeline {
         {
             stage 'Build images', {
                 checkout scm
-                sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.ci.yml build"
+                sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml build"
             }
 
             stage 'Project tests', {
                 try {
-                    sh "chmod 777 build/ && IMAGE_TAG=${commit} docker-compose -f docker-compose.ci.yml run --rm ci ./project_tests.sh"
+                    sh "docker run --name annotations_tests_${commit} elifesciences/annotations_ci:${commit}"
+                    sh "docker cp annotations_tests_${commit}:/srv/annotations/build/. build"
                     step([$class: "JUnitResultArchiver", testResults: 'build/phpunit.xml'])
```

Comment:
`cp` and archiving of results needs to be in `finally` to always be run, in order to be executed also on the failure of the tests
Can https://github.com/elifesciences/elife-jenkins-workflow-libs/blob/master/vars/dockerComposeProjectTests.groovy or https://github.com/elifesciences/elife-jenkins-workflow-libs/blob/master/vars/dockerProjectTests.groovy be used here?